### PR TITLE
Navigationbar service location

### PR DESCRIPTION
**Pull request by bennsimon**
Added service location on the navigation bar.

Check the following PR https://github.com/OpenSRP/opensrp-client-giz-malawi/pull/70

### CODE REVIEW

File: opensrp-giz-malawi/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -102,4 +102,5 @@
     <string name="action_start_sync">Sync Started</string>
     <string name="all_registers">All Registers</string>
     <string name="menu_child_clients">Child Clients</string>
+    <string name="due_only">Due Only</string>
```
Comment (ekigamba) [inline opensrp-giz-malawi/src/main/res/values/strings.xml]: Fix the position of the due Only Switch & Add the register name at the top of the register

Comment (ekigamba) [reply opensrp-giz-malawi/src/main/res/values/strings.xml]: Look at the ANC design for what I am talking about. Or countercheck with the actual design

Comment (bennsimon) [reply opensrp-giz-malawi/src/main/res/values/strings.xml]: updated on https://github.com/OpenSRP/opensrp-client-giz-malawi/pull/70

Comment (bennsimon) [reply opensrp-giz-malawi/src/main/res/values/strings.xml]: ok

**Pull request closed without merging**